packages/web/src/common/store/queue/sagas.ts

```diff
@@ -195,8 +198,7 @@ export function* watchPlay() {
             uid,
             isPreview,
             trackId: playActionTrack.track_id,
-            // Don't auto-advance after previews
-            onEnd: isPreview ? playerActions.stop : next
+            onEnd: next
```

Comment:
interesting, why did we do this before?

Reply:
i think previously we had decided not to play previews during autoplay in the lineup, but now we want to